bin/happo-cypress.js

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env node
+
+const http = require('http');
+const { spawn } = require('child_process');
+
+const makeRequest = require('happo.io/build/makeRequest').default;
+const compareReports = require('happo.io/build/commands/compareReports')
+  .default;
```

Comment:
I believe the happo.io package should have moved from devDependencies to dependencies https://github.com/happo/happo-cypress/blob/9cfc5aa00836e3352ea5e144f16b6579499e29cf/package.json#L18-L21

I did that in happo-e2e here: https://github.com/happo/happo-e2e/pull/65